.idea/workspace.xml

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It's good to remove the whole .idea/ directory from the repository and add it to .gitignore